international-journal-of-osteoarchaeology.csl

```diff
@@ -0,0 +1,248 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" default-locale="en-US" class="in-text" version="1.0" demote-non-dropping-particle="sort-only">
+  <info>
+    <title>International Journal of Osteoarchaeology</title>
+    <id>http://www.zotero.org/styles/international-journal-of-osteoarchaeology</id>
+    <link href="http://www.zotero.org/styles/international-journal-of-osteoarchaeology" rel="self"/>
+    <link href="http://onlinelibrary.wiley.com/journal/10.1002/%28ISSN%291099-1212/homepage/ForAuthors.html" rel="documentation"/>
+    <author>
+      <name>Piotr Jaskulski</name>
+      <email>piotr.jaskulski@gmail.com</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="anthropology"/>
+    <issn>1047-482X</issn>
+    <eissn>1099-1212</eissn>
+    <updated>2015-05-26T20:00:23+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="editor">
+    <names variable="editor" delimiter=", ">
+      <name and="text" initialize-with="" sort-separator=" " delimiter=", " name-as-sort-order="all"/>
+      <label form="short" prefix=" (" suffix=")" strip-periods="true"/>
+    </names>
+  </macro>
+  <macro name="anon">
+    <text term="anonymous" form="short" text-case="capitalize-first" strip-periods="true"/>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name name-as-sort-order="all" sort-separator=" " delimiter=", " form="long" delimiter-precedes-last="always" initialize-with=""/>
+      <label form="short" prefix=" (" suffix=")" strip-periods="true"/>
+      <substitute>
+        <names variable="editor"/>
+        <text macro="anon"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" and="symbol" delimiter=", " initialize-with=". "/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <text macro="anon"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="access">
+    <choose>
```

Comment:
would we really ever want both a DOI and a URL? You'd get this the way this is coded together with the DOI macro in line 240.